### DISCUSSION TITLE
Quality Menu on iOS

### DISCUSSION THREAD

**Discussion by kaambiz** [vidstack/player#895] [Q&A] [answered]
Hi, 
Quality menu is accessible everywhere except Safari on iOS, other parts of control are available though (I know, it wont appear on full screen anyway. I just want it for inline player). I made sure that preferNativeHLS is set true, still no show. Will it not work at all? Or am I doing something wrong as always?

Best Regards.

**Comment by mihar-22** [comment, marked as answer]
Hey! Unfortuantely, there is no API available today for us to control quality levels on iOS Safari. You're not doing anything wrong, there's just no ideal way to show a quality selection menu.
